cli/ballerina-tool/src/main/resources/cli-help/ballerina-compile.help

```diff
@@ -1,35 +1,33 @@
 NAME
-    Ballerina compile - Compiles Ballerina module(s) and generates shareable
+    Ballerina compile - Compile Ballerina module(s) and generate shareable
     .balo file(s).
```

Comment:
Double check this statement with the jBallerina team

Reply:
Removed the `ballerina-compile.help` file as it is not relevant now.